src/tests/multihost/alltests/test_misc.py

```diff
@@ -532,3 +541,80 @@ def test_0009_dbus_method_find_usrby_attr(multihost, backupsssdconf, ldap_posix_
             assert cmd2.stdout_text.strip('\n') in cmd.stdout_text, 'dbus is not fetching expected users'
         cmd1 = multihost.client[0].run_command(f'id -u {usr}@{domain_name}', raiseonerr=False)
         assert cmd1.stdout_text.strip('\n') not in cmd.stdout_text, 'dbus is fetching unwanted user'
+
+    @pytest.mark.tier1_4
+    def test_bz822236(self, multihost, backupsssdconf):
```

Comment:
The test does not use self. Please remove it and add @staticmethod decorator.

Reply:
Done